test/unit/gateways/quickpay_v10_test.rb

```diff
@@ -166,6 +166,17 @@ def test_transcript_scrubbing
     assert_equal scrubbed_transcript, @gateway.scrub(transcript)
   end
 
+  def test_test_url_and_version
+    gateway = ActiveMerchant::Billing::QuickpayV10Gateway.new(api_key: 'APIKEY')
```

Comment:
Please add a test similar to [this](https://github.com/activemerchant/active_merchant/blob/master/test/unit/gateways/quickpay_v10_test.rb#L31) where we check the version in the headers.